resources/views/settings/index.blade.php

```diff
@@ -83,8 +83,13 @@
               <div class="form-group">
                 <label for="name_order">{{ trans('settings.name_order') }}</label>
                 <select name="name_order" class="form-control">
-                  <option value="firstname_first" {{ (auth()->user()->name_order == 'firstname_first')?'selected':'' }}>{{ trans('settings.name_order_firstname_first') }}</option>
-                  <option value="lastname_first" {{ (auth()->user()->name_order == 'lastname_first')?'selected':'' }}>{{ trans('settings.name_order_lastname_first') }}</option>
+                  <option value="firstname_lastname" {{ (auth()->user()->name_order == 'firstname_lastname')?'selected':'' }}>{{ trans('settings.name_order_firstname_lastname') }}</option>
```

Comment:
It would have been more interesting to add a list [name_order kind => label] and iterate over it, no ?

Reply:
Done!